docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -145,7 +145,7 @@ There are four selectors, that define which queries run in which resource group:
   These are placed in a dynamically-created sub-group under the ``global.pipeline.tools`` group. The dynamic
   sub-group are created based on the named variable ``toolname``, which is extracted from the in the
```

Comment:
modification and grammar fix: (cc @mosabua)

```
The dynamic sub-groups are created based on the values of named variables ``toolname`` 
and ``user``. The values are derived from the source regular expression and the query user 
respectively.
```

Reply:
Done